src/System.Drawing.Primitives/src/System/Drawing/KnownColorTable.cs

```diff
@@ -1,358 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-namespace System.Drawing
-{
-    internal static class KnownColorTable
-    {
-        private static int[] s_colorTable;
-        private static string[] s_colorNameTable;
-
-        private static void EnsureColorTable()
-        {
-            // no need to lock... worse case is a double create of the table...
-            //
-            if (s_colorTable == null)
-            {
-                InitColorTable();
-            }
-        }
-
-        private static void InitColorTable()
-        {
-            int[] values = new int[KnownColor.LastColor - KnownColor.FirstColor + 1];
-
-            // just consts...
-            //
-            values[(int)KnownColor.Transparent] = 0x00FFFFFF;
```

Comment:
This code should be persisted in the common source file, and should be active when `!FEATURE_WINDOWS_SYSTEM_COLORS`, otherwise we have a behavior regression on uap in System.Drawing.Primitives.